lib/Lib.Sequence.Lemmas.fsti

```diff
@@ -98,6 +98,18 @@ val repeati_right_extensionality:
     Loops.repeat_right 0 n (Loops.fixed_a a) f acc0 ==
     Loops.repeat_right lo_g (lo_g + n) (Loops.fixed_a a) g acc0)
 
+/// A specialized version of the lemma above, for only shifting one computation,
+/// but specified using repeati instead
```

Comment:
I know it's not your code, but why is the lemma above stated in terms of repeat_right when its name implies it's a lemma related to repeati?

Reply:
I wondered the same thing. My best guess is that the name comes from the left part of the equality, which is the same as repeati n f acc0 (which can be derived using LoopCombinators.repeati_def). We could maybe replace the left part directly by a repeati